recipes/pytest-catchlog/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pytest-catchlog" %}
+{% set version = "1.2.2" %}
+{% set hash_type = "sha256" %}
+{% set hash = "4be15dc5ac1750f83960897f591453040dff044b5966fe24a91c2f7d04ecfcf0" %}
+{% set ext = "zip" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ ext }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ ext }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytest
+
+test:
+  requires:
+    - pytest
+  imports:
+    - pytest_catchlog
+  commands:
+    - py.test --help
+
+about:
+  home: https://github.com/eisensheng/pytest-catchlog
+  license: MIT
+  summary: py.test plugin to catch log messages. This is a fork of pytest-capturelog.
+
+extra:
+  recipe-maintainers:
+    - nicoddemus
```

Comment:
@msarahan since you submitted #1360 do you want to be added here? Also, can you review and merge this one :wink: